addons/editor-stage-left/addon.json

```diff
@@ -13,13 +13,14 @@
     }
   ],
   "dynamicDisable": true,
+  "dynamicEnable": true,
   "userstyles": [
     {
       "url": "stageleft.css",
       "matches": ["https://scratch.mit.edu/projects/*"]
     }
   ],
-  "tags": ["editor", "theme"],
+  "tags": ["editor"],
```

Comment:
This addon was always in the themes category, this would remove it.

Reply:
@mxmou See changes for settings page, the category was hardcoded to "theme" for this addon, even tho it lacks the tag.